packer/install-dependencies.sh

```diff
@@ -156,7 +156,7 @@ for TILLER_VERSION in ${TILLER_VERSIONS}; do
     echo "  - ${CONTAINER_IMAGE}" >> ${RELEASE_NOTES_FILEPATH}
 done
 
-CLUSTER_AUTOSCALER_VERSIONS="1.13.1 1.12.2 1.3.7 1.3.4 1.3.3 1.2.2 1.1.2"
+CLUSTER_AUTOSCALER_VERSIONS="1.13.1 1.13.2 1.12.2 1.12.3 1.3.7 1.3.8 1.3.4 1.3.3 1.2.2 1.2.5 1.1.2"
```

Comment:
Could we sort these autoscaler versions in descending order? I find this confusing to read.

`1.13.2 1.13.1 1.12.3 1.12.2 1.3.8 1.3.7 1.3.4 1.3.3 1.2.5 1.2.2 1.1.2`